easybuild/tools/github.py

```diff
@@ -684,6 +727,38 @@ def _easyconfigs_pr_common(paths, start_branch=None, pr_branch=None, target_acco
     return file_info, git_repo, pr_branch, diff_stat
 
 
+def scan_all_easyconfigs(patch_name):
+    """
+    Scan all easyconfigs in the robot path to determine which software a patch file belongs to
+
+    @param patch_name: name of the patch file
+    """
+    robot_path = build_option('robot_path')[0]
+    soft_name = ""
+    found = False
+    counter = 0
+    for (dirpath, _, filenames) in os.walk(robot_path):
+        if found:
+            break
+        for filename in [fn for fn in filenames if ".eb" in fn]:
+            rawtxt = read_file(os.path.join(dirpath, filename))
+            if "patches" in rawtxt and "TEMPLATE" not in filename:
```

Comment:
TEMPLATE has the "patches"  parameter, so it will match, but can't be parsed as an EasyConfig in the line below
i could change the check to `patches = [.+]` to make sure it's not an empty list?